StudentDetails.java

```diff
@@ -0,0 +1,155 @@
+import java.util.*;
+import java.util.Scanner;
+
+class StudentData {
+    private int sno;
+    private String sfname;
+    private String slname;
+
+    StudentData(int sno, String sfname, String slname) {
```

Comment:
Why usage of a default access modifier? Also keep an eye on using appropriate namings -> avoid abbreviations whenever possible to minimize confusion!